src/concrete/ml/sklearn/base.py

```diff
@@ -1171,7 +1172,9 @@ def compile(
     def _inference(self, q_X: numpy.ndarray) -> numpy.ndarray:
         self.check_model_is_fitted()
 
-        return self.quantized_module_.quantized_forward(q_X)
+        result = self.quantized_module_.quantized_forward(*[q_X])
```

Comment:
I don't think this is necessary right ? `quantized_forward` should handle single-input fine !

Reply:
indeed!